### PR TITLE
NPA-3887 Document Sandbox Responses

### PR DESCRIPTION
**Pull request by JackPlowman**
## Ticket Link

https://nhsd-jira.digital.nhs.uk/browse/NPA-3887

## Description/Change Summary

- Documented Related Person Sandbox Responses in schema
- Flipped schema nhs numbers for proxy and patient to match sandbox examples
- Fixed incorrect sandbox response for invalid identifier
- Added invalid system response

# How to test?

- Sandbox responses test evidence - https://nhsd-jira.digital.nhs.uk/browse/NPA-3887?focusedId=2177613&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2177613

### CODE REVIEW

**Review by LeoKHoward — APPROVED** [review]

**Comment by JackPlowman** [comment]
Do not merge PR until @miiisterjim has approved the PR

**Comment by JackPlowman** [comment]
Closing PR and created second PR to with rebased branch
https://github.com/NHSDigital/validated-relationships-service-api/pull/113

**Pull request closed without merging**
